js/index.js

```diff
@@ -17,22 +21,95 @@ const skills = [
     'Engineering', 
     'Reserch'
 ];
+//Fill out a Skill section with skills from the skills array. 
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul');
 for(let i = 0; i < skills.length; i++) {
     let skill = document.createElement('li');
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
-const listItems = skillsSection.getElementsByTagName('li');
+//Add an event to the Skill section
+/* const listItems = skillsSection.getElementsByTagName('li');
 for(let i = 0; i < listItems.length; i++) {
```

Comment:
Make sure delete unnecessary code. If you've commented out code because it doesn't work for you and you have refactored and still don't need it, make sure you delete that unnecessary code.

Reply:
> Make sure delete unnecessary code. If you've commented out code because it doesn't work for you and you have refactored and still don't need it, make sure you delete that unnecessary code.

ok, thanks, deleted